sanic/app.py

```diff
@@ -835,6 +835,11 @@ def run(
         access_log=True,
         **kwargs
     ):
+        try:
+            assert "loop" not in kwargs
```

Comment:
Rather than do it as an `assert` statement (which can be turned off), I think it would be better just to do:

```
if "loop" in kwargs:
```